client/src/app/core/repositories/meeting-model-with-error-repository.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionType } from '../core-services/action.service';
+import { BaseModel, ModelConstructor } from 'app/shared/models/base/base-model';
+import { BaseViewModel } from 'app/site/base/base-view-model';
+import { ErrorService } from '../ui-services/error.service';
+import { MeetingModelBaseRepository } from './meeting-model-base-repository';
+import { RepositoryServiceCollector } from './repository-service-collector';
+
+export abstract class MeetingModelWithErrorRepository<
```

Comment:
Wh did you add this class? Do you try to avoid the repeating `.catch(this.raiseError)` in all components?